### PR TITLE
updated WP content

### PR DESCRIPTION
**Pull request by sofyalaski**
I updated the description of WP 1 and 3 a little and added some links. Please tell me what you think or also change. 

### WP1 
task 1 description updated:
* added bullet points, because the text wasn't actually a text I think? 
* added some changes about oscem ( not a draft anymore, added link)
* Commented out "Conversion tools under development for major instrument manufacturers and formats" does that only include WP3 or something else? You can put it back, but it needs clarification which formats/manufactures meant. 

### WP3
* added more info and links to task 1
* task2 now has a little more details. @despadam you are very welcome to add other sources.

### CODE REVIEW

File: deliverable-wp1.md [hunk 1]
```diff
@@ -54,12 +54,12 @@ The Metadata Standards work package contains the following tasks and components.
...
 #### Metadata-Standards-1: Community consensus and commitment to data collection standards ####
 
-Established the <a href="https://github.com/osc-em">Open Standards Community for EM</a>
-Workshop 22-23 Feb 2024 with participants from facilities, software, and repositories
-Draft schema available for EM metadata. The goal is to include metadata required for future processing and deposition.
-Schema terms are defined by existing ontologies where available: CryoEM ontology, PDBx/mmCIF dictionary, Helmholz EM Glossary, NeXus-FAIRmat NXem format
-Conversion tools under development for major instrument manufacturers and formats
-Coordinate with PREMISE project regarding terms and ontologies
+- Established the <a href="https://github.com/osc-em">Open Standards Community for EM</a>
+Workshop 22-23 Feb 2024 with participants from facilities, software developers and repositories curators.
```
Comment (sbliven) [inline deliverable-wp1.md:58]: Dropped a bullet symbol here

File: deliverable-wp1.md [hunk 2]
```diff
@@ -54,12 +54,12 @@ The Metadata Standards work package contains the following tasks and components.
...
+- Schema is now <a href="https://osc-em.github.io/OSCEM_Schemas/">available</a> for EM metadata. During next milestones, metadata required for processing will be included by our collaborators and support for Material Sciences will be extended. 
+- Schema terms are defined with respect to existing ontologies where available: CryoEM ontology, PDBx/mmCIF dictionary, Helmholz EM Glossary, NeXus-FAIRmat NXem format.
+<!-- Conversion tools under development for major instrument manufacturers and formats -->
```
Comment (sbliven) [inline deliverable-wp1.md:61]: Why commented? Let's rather day 'tools available for ... formats (Thermo Fisher EPU and SerialEM)'

**Review by sbliven — CHANGES_REQUESTED** [review]
Doesn't this need to update the site data too to mark 3.2 as 'Done'?

File: deliverable-wp1.md (hunk 1 above)
Comment (sofyalaski) [reply deliverable-wp1.md:58]: oh thanks

File: deliverable-wp1.md (hunk 2 above)
Comment (sofyalaski) [reply deliverable-wp1.md]: yes, the clarification was missing for me. I added it back.

**Comment by sofyalaski** [comment]
@sbliven, sorry,  I am a bit lost. Did you mean repository integration task2? Because I think I would then shift it to "in progress". Me and Despina talked today and thought maybe it's too early to say we are not going to implement any automatic deposition for material sciences. The way it's written now is probably not the best and I will add a sentence that we are still planning to implement something. In the next milestone, both Me and Despina will try to find out more about these and other databases. 

Anyway, I first thought your comment was about WP1 task 3, that the web-page is now working. So I added a bit more complete sentences. If that's not correct, I can revert.

**Review by despadam — CHANGES_REQUESTED** [review]

**Comment by sofyalaski** [comment]
I also wanted to change the figure that is cited in WP2. @sbliven, do you have it somewhere in a better resolution?

**Review by ytittes — APPROVED** [review]

**Review by despadam — APPROVED** [review]

**Comment by sbliven** [comment]
@sofyalaski 
> Did you mean repository integration task2? Because I think I would then shift it to "in progress". Me and Despina talked today and thought maybe it's too early to say we are not going to implement any automatic deposition for material sciences.

👍 'In progress' is fine

> I also wanted to change the figure that is cited in WP2. @sbliven, do you have it somewhere in a better resolution?

I can't find the source. Best resolution is 1317 × 761: [230711 scicat overview.png](https://drive.google.com/open?id=12lfy8BKvImjd4_d6hdvQKr7OPoyNHFJw&usp=drive_fs)

**Pull request merged**